mala/network/multi_predictor.py

```diff
@@ -0,0 +1,342 @@
+"""Tester class for testing a network."""
+import glob
+import os
+from zipfile import ZipFile, ZIP_STORED
+
+import ase.io
+try:
+    import horovod.torch as hvd
+except ModuleNotFoundError:
+    # Warning is thrown by Parameters class
+    pass
+import numpy as np
+import torch
+
+from mala.common.parallelizer import printout, get_rank, barrier
+from mala.network.predictor import Predictor
+from mala.network.network import Network
+from mala.datahandling.data_scaler import DataScaler
+from mala.datahandling.data_handler import DataHandler
+from mala import Parameters
+
+
+class MultiPredictor(Predictor):
```

Comment:
I'd call it `SimpleEnesmblePredictor`. I have no super strong attachment to the `SimpleEnesmble` name as such. We should just

* make sure to clarify that this is *not* the method termed Deep Ensemble from https://proceedings.neurips.cc/paper/2017/hash/9ef2ed4b7fd2c810847ffa5fa85bce38-Abstract.html / https://arxiv.org/abs/1612.01474 [a poor name btw, but that's another story], since we do not learn a noise model but just have the epistemic uncertainty part from a .. well .. simple ensemble of models
* have the same method name here and in the ASE interface

Reply:
Yeah, that makes sense. I like simple ensemble and renamed the class accordingly.